src/App.tsx

```diff
@@ -1,61 +1,118 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import { User } from './types/Types';
+import { Todo } from './types/Types';
+import todosFromServer from './api/todos';
+import React, { useState } from 'react';
 
 export const App = () => {
+  const initialTodos: Todo[] = todosFromServer.map(todo => ({
+    ...todo,
+    user: usersFromServer.find(u => u.id === todo.userId) as User,
```

Comment:
The use of `as User` for type assertion assumes that the `find` method will always return a user. If no user is found, this will result in `undefined`, which could lead to runtime errors. Consider handling the case where a user might not be found.